src/constants/colors.ts

```diff
@@ -0,0 +1,12 @@
+export const colors = {
+    PRIMARY: {
+      black: '#000000',
+      gray: '#979797',
```

Comment:
change the names of the color to upper letters

Reply:
Done